src/index.html

```diff
@@ -10,13 +10,106 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
```

Comment:
HTML Formatting: The `meta` tag has two attributes on one line. For better readability, it's recommended to start each attribute on a new line with a 2-space indentation.